src/main/java/io/github/externschool/planner/service/TeacherServiceImpl.java

```diff
@@ -87,4 +89,28 @@ public void deleteTeacherById(Long id) {
             teacherRepository.deleteById(id);
         }
     }
+
+    @Override
+    @Scheduled(cron = "0 0 01 ? * SAT")
+    public void updateTeacherSchedule(){
+
+        List<Teacher> teachers = findAllTeachers();
+
+        for (Teacher teacher: teachers
+             ) {
+            for (int i = 0; i < 5; i++) {
+
+                List<ScheduleEvent> events = scheduleService
+                                            .getEventsByOwnerAndDate(teacher.getVerificationKey().getUser()
+                                            , Constants.FIRST_MONDAY_OF_EPOCH.plusDays(i));
```

Comment:
comma has to be in the previous line. and i'd format putting every new parameter in a new line, like this:
```
List<ScheduleEvent> events = scheduleService.getEventsByOwnerAndDate(
            teacher.getVerificationKey().getUser(),
            Constants.FIRST_MONDAY_OF_EPOCH.plusDays(i));
```